themes/default/layouts/page/get-started.html

```diff
@@ -0,0 +1,57 @@
+{{ define "hero" }}
+    {{ partial "hero.html" (dict "title" .Params.title) }}
+{{ end }}
+
+{{ define "main" }}
+    <section class="container mx-auto mt-20">
+        <div class="shape-background get-started-install-section">
+            <div class="shape-container">
+                <div class="circle-right"></div>
+            </div>
+        </div>
+        <div class="text-center mb-8 bg-violet-200 w-12 h-12 rounded-full mx-auto">
+            <p class="inline-block m-1">
+                <span class="text-violet-600 text-3xl font-display font-bold">1</span>
+            </p>
+        </div>
+        <h2 class="text-center">Install the Pulumi CLI</h2>
+        <p class="text-center">
+            <span class="max-w-md inline-block">To use open source Pulumi SDK or the Pulumi Console UI you need to have the Pulumi CLI installed.</span>
+        </p>
+        <div class="mt-12 mb-6 get-started-install">
+            <pulumi-install></pulumi-install>
+        </div>
+        <p class="text-center">Or review <a href="{{ relref . "/docs/get-started/install" }}" class="underline">other installation options</a>.</p>
+    </section>
+    <section class="container mx-auto mt-20">
+        <div class="shape-background get-started-step-2-section">
+            <div class="shape-container">
+                <div class="circle-left"></div>
+            </div>
+        </div>
+        <div class="text-center mb-8 bg-salmon-200 w-12 h-12 rounded-full mx-auto">
+            <p class="inline-block">
+                <span class="text-salmon-600 text-3xl font-display font-bold">2</span>
+            </p>
+        </div>
+        <h2 class="mb-8 text-center">Start learning or explore</h2>
+
+        <div class="lg:flex items-stretch mb-20">
+            <div class="w-full lg:w-1/2 px-4 py-2 lg:p-8 lg:px-4">
+                <div class="lg:shadow-2xl pl-12 pr-12 pb-12 pt-8 g:pb-12 rounded-xl">
+                    <h3><span class="text-2xl">Start learning</span></h4>
+                    <p class="mb-10">Pick a cloud and a language and deploy your first application using open source Pulumi!</p>
+                    <a href="{{ relref . "/docs/get-started" }}" class="btn-primary inline-block">Follow the Getting Started Tutorial</a>
+                </div>
+            </div>
+
+            <div class="w-full lg:w-1/2 px-4 py-2 lg:p-8 lg:px-4">
+                <div class="lg:shadow-2xl pl-12 pr-12 pb-12 pt-8 g:pb-12 rounded-xl bg-white">
+                    <h3><span class="text-2xl">Explore</span></h4>
+                    <p class="mb-10">The optional Pulumi Console UI manages deployment state and provides collaboration tools. Sign in to explore!</p>
```

Comment:
Suggestion from Christian

```suggestion
                    <p class="mb-10">The optional Pulumi Console manages deployment state and provides collaboration tools. Sign in to explore!</p>
```